src/env.h

```diff
@@ -422,6 +423,47 @@ enum class DebugCategory {
   CATEGORY_COUNT
 };
 
+class AccessControl {
+ public:
+  enum Permission {
```

Comment:
any reason this is defined as an `enum` and not `enum class`?

Reply:
@gioragutt Nothing, besides the fact that it creates an extra scope inside what is already a scope, so accessing it means a bit more stuff to write out.

(One could move it one layer up into the `node` namespace, but … feels right here to me?)

Reply:
Okay, I guess that since this is not the global scope, this is fine.
I just have traumas from working with global scoped non-class enums 😅